src/analysis/souffle/sensitivity_analysis_unit_test_interface.dl

```diff
@@ -0,0 +1,16 @@
+#ifndef SENSITIVITY_ANALYSIS_UNIT_TEST_INTERFACE_DL_
+#define SENSITIVITY_ANALYSIS_UNIT_TEST_INTERFACE_DL_
+
+#include "src/analysis/souffle/sensitivity_analysis.dl"
+
+//An interface used for unit testing of the sensitivity analysis. 
+//Only outputs value sensitivities
+
+.input isOperation(delimiter=";")
+.input isInitialInput(delimiter=";")
+.input expectedResult(delimiter=";")
+.output actualResult(delimiter=";")
+.output expectedResult(delimiter=";")
```

Comment:
Why is `expectedResult` both an input and an output relation?

Reply:
It was an attempt to fit in with the existing bazel rule. I reused another one written by you and it doesn't have this problem anymore.